trials.js

```diff
@@ -0,0 +1,49 @@
+"use strict"; //Strict mode turn on
```

Comment:
It could be better if you delete these irrelevant trials.js pages from your code setup.😊

Reply:
As suggested by the previous reviewer, kindly consider explaining why the changes you're requesting the student make is important.